ci/config-gmt-windows.sh

```diff
@@ -12,6 +12,8 @@ set (DCW_ROOT "$ENV{COASTLINEDIR}/dcw")
 set (GMT_ENABLE_OPENMP TRUE)
 
 set (GMT_DLL_RENAME "gmt_w64")
+set (PSL_DLL_RENAME "postscriptlight_w64")
+set (SUPP_DLL_RENAME "supplements_w64")
```

Comment:
I don't think this is correct. Those are not fixed names. They will be ``postscriptlight_w32`` for 32 bits builds

Reply:
``PSL_DLL_RENAME`` are variables set by me in my ConfigUser.cmake

Reply:
They're only for the CI jobs. Currently, our CI jobs are building 64-bit only.